docs/content/docs/install/settings.md

```diff
@@ -186,55 +186,95 @@ A few settings are available to configure this feature:
+  variable:
 
-* `{{ namespace }}`: The target namespace where the pipelinerun is executed
-* `{{ pr }}`: The PipelineRun name.
+  * `{{ namespace }}`: The target namespace where the pipelinerun is executed
+  * `{{ pr }}`: The PipelineRun name.
+  
+  example: `https://mycorp.com/ns/{{ namespace }}/pipelinerun/{{ pr }}`
 
- example: `https://mycorp.com/ns/{{ namespace }}/pipelienrun/{{ pr }}`
+  Moreover it can access the [custom parameters](../guide/repositorycrd/#custom-parameter-expansion) from a
+  Repository CR. For example if the user has a parameter in their Repo CR like this :
 
-* `custom-console-url-pr-tasklog`
+   ```yaml
+   [...]
+   spec:
+    params:
+      - name: custom
+        value: value
+   ```
 
- Set this to the URL where to view the log of the taskrun of the `PipelineRun`. This is
- shown when we post a result of the task breakdown to link to the logs of the taskrun.
+  and the global configuration setting for `custom-console-url-pr-details` is:
+  
+  `https://mycorp.com/ns/{{ namespace }}/{{ custom }}`
+  
+  the `{{ custom }}` tag in the URL is expanded as `value`.
+  
+  This let operator to add specific informations like a `UUID` about a user as
+  parameter in their repo CR and let it link to the console.
+  
+* `custom-console-url-pr-tasklog`
+  
+  Set this to the URL where to view the log of the taskrun of the `PipelineRun`. This is
+  shown when we post a result of the task breakdown to link to the logs of the taskrun.
 
- The URL suports templating for these value:
+  The URL suports custom parameter on Repo CR and the standard parameters as
+  described in the `custom-console-url-pr-details` setting and as well those added
+  values:
 
-* `{{ namespace }}`: The target namespace where the pipelinerun is executed
-* `{{ pr }}`: The PipelineRun name.
-* `{{ task }}`: The Task name in the PR
-* `{{ pod }}`: The Pod name of the TaskRun
-* `{{ firstFailedStep }}`: The name of the first failed step in the TaskRun
+  * `{{ namespace }}`: The target namespace where the pipelinerun is executed
+  * `{{ pr }}`: The PipelineRun name.
+  * `{{ task }}`: The Task name in the PR
+  * `{{ pod }}`: The Pod name of the TaskRun
+  * `{{ firstFailedStep }}`: The name of the first failed step in the TaskRun
 
- example: `https://mycorp.com/ns/{{ namespace }}/pipelinerun/{{ pr }}/logs/{{ task }}#{{ pod }}-{{ firstFailedStep }}`
+  example: `https://mycorp.com/ns/{{ namespace }}/pipelinerun/{{ pr }}/logs/{{ task }}#{{ pod }}-{{ firstFailedStep }}`
 
 ## Pipelines-As-Code Info
 
   There are a settings exposed through a config map for which any authenticated
-  user can access to know about the Pipeline as Code status.
+  user can access to know about the Pipeline as Code status. This Configmap
+  will be automatically created with the [OpenShift Pipelines Operator](https://docs.openshift.com/container-platform/latest/cicd/pipelines/understanding-openshift-pipelines.html)
```

Comment:
I think it would be nice to mention the name of the ConfigMap, seeing as the user might want to access it?